### PR TITLE
804 feature request sqlninja tool implementation

### PR DESCRIPTION
**Pull request by connorwjordan**
Sqlninja is a tool that can exploit SQL injection vulnerabilities on web applications that use Microsoft SQL Server as back end.

Implemented 4 modes in a dropdown to pick, ability to set filepath for configuration file otherwise tool has a default, and ability to enable verbose output.

### CODE REVIEW

File: src/components/Sqlninja/sqlninja.tsx [hunk 1]
```diff
@@ -0,0 +1,225 @@
+import { Button, NativeSelect, Stack, TextInput, Switch } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { useCallback, useState, useEffect } from "react";
+import { CommandHelper } from "../../utils/CommandHelper";
+import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
```
Comment (Oliver2049) [inline src/components/Sqlninja/sqlninja.tsx:6]: You can use LoadingOverlayAndCancelButtonPkexec 

```suggestion
import { LoadingOverlayAndCancelButtonPkexec } from "../OverlayAndCancelButton/OverlayAndCancelButton";
```

File: src/components/Sqlninja/sqlninja.tsx [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+                        label={"Configuration File Path"}
+                        placeholder="default: sqlninja.conf"
+                        {...form.getInputProps("filePath")}
+                    />
+                    <Switch
+                        label="Verbose Mode"
+                        checked={checkedVerbose}
+                        onChange={(e) => setCheckedVerbose(e.currentTarget.checked)}
+                    />
+                    <Button type={"submit"}>Start</Button>
```
Comment (Oliver2049) [inline src/components/Sqlninja/sqlninja.tsx:216]: You should use Start {title}
```suggestion
                    <Button type={"submit"}>Start {title}</Button>
```

**Comment by Oliver2049** [comment]
Hi @connorwjordan 
Overall, this is very good and you just need to add a few changes and it will be perfect.
Regards,
Oliver

**Comment by michaeljpigott** [comment]
@connorwjordan, this is still waiting for you to act on the requested changes. 

Kind regards

Michael

**Comment by connorwjordan** [comment]
@Oliver2049 Hi thanks for the review, I have committed all requested changed.

**Review by Oliver2049 — APPROVED** [review]
Hi @connorwjordan 
Overall, it looks very good to me. Thank you for your effort.
Regards
Oliver

**Pull request merged**